pom.xml

```diff
@@ -467,6 +474,14 @@ under the License.
     </build>
 
     <profiles>
+        <profile>
+            <id>flink-1.14</id>
+            <properties>
+                <flink.version>1.14.4</flink.version>
```

Comment:
Is it required to still have 1.14.x support?
Why just not to update to 1.15.x?
Or otherwise somehow need to guarantee that future changes will not add some 1.15.x specific api which could break 1.14.x usage... It seems currently ci does not check this profile... or did I miss something?

Reply:
Thanks for your option, @snuyanzin. I have removed this profile to have a try.